tests/Doctrine/Tests/DBAL/Platforms/SqlitePlatformTest.php

```diff
@@ -744,4 +744,19 @@ public function testQuotesTableNameInListTableForeignKeysSQL()
     {
         $this->assertContains("'Foo''Bar\\'", $this->_platform->getListTableForeignKeysSQL("Foo'Bar\\"), '', true);
     }
+
+    public function testSupportsIntAndStringInDateArithmeticExpressions()
```

Comment:
Just a note: probably it's a good idea to rename this to testSupportsIntAnd**FieldReference**InDateArithmeticExpressions